trieste/acquisition/optimizer.py

```diff
@@ -117,8 +128,8 @@ def optimize_discrete(space: DiscreteSearchSpace, target_func: AcquisitionFuncti
 
 def generate_continuous_optimizer(
     num_initial_samples: int = NUM_SAMPLES_MIN,
-    num_optimization_runs: int = 1,
-    num_recovery_runs: int = 5,
+    num_optimization_runs: int = 10,
```

Comment:
10 seems a bit much, I would expect 3-5 to be enough, did you test it in any way - before introducing a lot more computation by default?

Reply:
Its hard to know how many are enough. I tested this change to see if it slowed anything down, and it didnt seem to! Happy to dial it back to 5 if you want?

Reply:
thats strange that it didn't slow things down - that might suggest additional run is not the bottleneck but the evaluations, perhaps its fine as it is

actually, more relevant default is the automatic selector one, not this one - in a 10D user would have 100 runs, perhaps necessary but that definitely seems a lot